# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/zzz/hotfix-ln-sf.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- zzz-0.2.0/Makefile.vanilla	2026-09-08 21:00:44.118113204 +0200
+++ zzz-0.2.0/Makefile	2026-09-08 21:02:21.034935932 +0200
@@ -9,7 +9,7 @@
 
 ASCIIDOCTOR   := asciidoctor
 INSTALL       := install
-LN_S          := ln -s
+LN_S          := ln -sf
 GIT           := git
 SED           := sed
 
